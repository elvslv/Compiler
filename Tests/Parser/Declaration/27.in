type
 Tmatrix = array[1..1] of integer;
var
 Matrix: ^Tmatrix;
 Sums: ^Tmatrix;
 Nums: ^Tmatrix;
 i,j,k1,k2,m,n: integer;
