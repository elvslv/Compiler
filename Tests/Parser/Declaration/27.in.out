type TMATRIX = array [const 1..const 1] of INTEGER;
var MATRIX : ^TMATRIX;
var SUMS : ^TMATRIX;
var NUMS : ^TMATRIX;
var I : INTEGER;
var J : INTEGER;
var K1 : INTEGER;
var K2 : INTEGER;
var M : INTEGER;
var N : INTEGER;
